src/migrations/20241108102623-GrantRelationshipToActive.js

```diff
@@ -0,0 +1,46 @@
+const { prepMigration } = require('../lib/migration');
+
+const goalText = '(Monitoring) The recipient will develop and implement a CAP/QIP to address monitoring findings.';
```

Comment:
That is not the posted standard goal
```suggestion
const goalText = '(Monitoring) The recipient will develop and implement a QIP/CAP to address monitoring findings.';
```

Reply:
This was copied directly from the JIRA ticket.